etc/bento.env

```diff
@@ -272,15 +274,12 @@ BENTOV2_FEDERATION_CPUS=2
 
 # Event-Relay
 BENTO_EVENT_RELAY_REPO=https://github.com/bento-platform/bento_event_relay
-BENTOV2_EVENT_RELAY_TAG_OR_BRANCH=v1.0.0
-BENTOV2_EVENT_RELAY_BASE_IMAGE=node
-BENTOV2_EVENT_RELAY_BASE_IMAGE_VERSION=15.5.1-alpine3.10
-BENTOV2_EVENT_RELAY_IMAGE=bentov2-event-relay
-BENTOV2_EVENT_RELAY_VERSION=0.0.1
+BENTOV2_EVENT_RELAY_IMAGE=ghcr.io/bento-platform/bento_event_relay
+BENTOV2_EVENT_RELAY_VERSION=latest
```

Comment:
maybe good to pin version like we were doing for Git tag? in this case can use 2.0.1 as the tag - that way we control the version with releases to bentov2